src/Models/Timezone.php

```diff
@@ -7,15 +7,11 @@
 use Tipoff\Addresses\Transformers\TimezoneTransformer;
 use Tipoff\Support\Contracts\Addresses\TimezoneInterface;
 use Tipoff\Support\Models\BaseModel;
-use Tipoff\Support\Traits\HasCreator;
 use Tipoff\Support\Traits\HasPackageFactory;
-use Tipoff\Support\Traits\HasUpdater;
 
 class Timezone extends BaseModel implements TimezoneInterface
 {
     use HasPackageFactory;
```

Comment:
I'm assuming you double checked that we don't need these fields

Reply:
Yes, the timezone table doesn't have created_at, updated_at or creator_id, updater_id. The alternative is to add those to the table and revert these changes.